packages/framework-provider-local-infrastructure/src/index.ts

```diff
@@ -48,7 +48,7 @@ export const Infrastructure = (rocketDescriptors?: RocketDescriptor[]): Provider
       router.use('/graphql', new GraphQLController(graphQLService).router)
       if (rockets && rockets.length > 0) {
         rockets.forEach((rocket) => {
-          rocket.mountStack(config, router)
+          rocket.mountStack(config, router, port)
```

Comment:
It could be possible that we need to pass more specific parameters in the future. In order to avoid any possible breaking changes, I would pass a configuration record as the third parameter, where right now we only pass the `port` field. Something like:
```suggestion
          rocket.mountStack(config, router, { port })
```

Reply:
I agree. We are missing a configuration record here! 

We would have the same problem now with the `start` method:

```typescript
 start: async (config: BoosterConfig, port: number): Promise<void> => {
```

Expanding the `start` functionality already implies a potential breaking change

Reply:
Done